canonical-web-and-design/snapcraft.io
# Implement notification for dropped architectures

## Done
- Implement notification for i386 when the publisher switches the base to core20 or higher

## How to QA
- Visit this snap, it was using core18 and the last builds were for i386. Now it uses core20, the message should appear:
  https://snapcraft-io-4030.demos.haus/test-snap-fran/builds
- Visit dotrun which uses core20 now but many builds happened since the last build for i386, the message shouldn't appear:
  https://snapcraft-io-4030.demos.haus/dotrun/builds
- Visit this snap which is using core18, the message shouldn't appear:
  https://snapcraft-io-4030.demos.haus/testfran3/builds

Because we are in the demo environment you will get this error before:
![image](https://user-images.githubusercontent.com/6353928/173589838-017fbb9b-967a-46ec-8699-1f89226fddf7.png)
Click on the link and authorize the staging GitHub app, once done you can QA again.

## Issue / Card
Fixes https://github.com/canonical-web-and-design/marketplace-tribe/issues/2589

## Screenshots
![image](https://user-images.githubusercontent.com/6353928/173578170-f886b280-9f53-4963-bc03-5f392e411e55.png)

## Todo after approval
- [x] Revert callback URL for staging GitHub application to `https://staging.snapcraft.io/github/auth/verify`
